### ISSUE TITLE
pypi package?

### ISSUE THREAD

**Issue by seantalts** [adamhaber/stan2tfp#2]
Would be nice to have this up on pypi!

**Comment by adamhaber** [comment]
Of course! :-) I wanted to make sure we have at least a reasonable coverage of Stan distributions before uploading to pip. I'll submit a PR with a couple of more distributions next week, improve the docs, and then I'll upload it.
